registrations/serializers.py

```diff
@@ -159,6 +159,26 @@ class Meta:
         model = SignUp
 
 
+class RegistrationUserIdField(serializers.PrimaryKeyRelatedField):
+    def get_queryset(self):
+        registration_id = self.context["request"].parser_context["kwargs"]["pk"]
+        return RegistrationUser.objects.filter(registration__pk=registration_id)
```

Comment:
Why does this exist? I don't quite get it 😅

Reply:
The reason to have RegistrationUserIdField class is to validate registration user id values. They have to exist and to be in same registration that user is updating. So the user cannot accidentally edit registration_users of other registration and move them to the registration which he is updating.

Reply:
Makes sense.